### PR TITLE
Probable Condescending Language Removed

### PR DESCRIPTION
**Pull request by mananchawla2005**

### CODE REVIEW

File: _documentation/en/telephony/code-snippets/call/delete-call-resource.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ navigation_weight: 1
 
 # Delete a call resource
 
-This example shows you how to delete a call. If the call is active, this SHOULD hang up the call. #TODO
```
Comment (lukeocodes) [inline _documentation/en/telephony/code-snippets/call/delete-call-resource.md:8]: Don't agree with this one!

File: _documentation/en/telephony/code-snippets/call_leg/delete-call-leg.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ navigation_weight: 1
 
 # Delete a call leg
 
-This example shows you how to delete a call leg. If the call is active, this SHOULD hang up the call. #TODO
```
Comment (lukeocodes) [inline _documentation/en/telephony/code-snippets/call_leg/delete-call-leg.md:8]: This one too!

File: _documentation/en/vonage-integration-platform/concepts/call-events.md [hunk 3]
```diff
@@ -22,11 +22,11 @@ All events representing the state of telephone calls share common fields and mea
 
 ## Event States
 
-The VIS APIs present a simple event state model for telephone calls, hiding the differences and complexities of either Vonage Business Communications and Vonage Enterprise. These event states are delivered in response to method APIs (such as place call, get calls) and included in webhook event notifications. Understanding the event state model is critical to building external integrations.
+The VIS APIs present a event state model for telephone calls, hiding the differences and complexities of either Vonage Business Communications and Vonage Enterprise. These event states are delivered in response to method APIs (such as place call, get calls) and included in webhook event notifications. Understanding the event state model is critical to building external integrations.
```
Comment (lukeocodes) [inline _documentation/en/vonage-integration-platform/concepts/call-events.md:25]:
```suggestion
The VIS APIs present an event state model for telephone calls, hiding the differences and complexities of either Vonage Business Communications and Vonage Enterprise. These event states are delivered in response to method APIs (such as place call, get calls) and included in webhook event notifications. Understanding the event state model is critical to building external integrations.
```

**Review by lukeocodes — CHANGES_REQUESTED** [review]
Thanks for this. I think these types of PRs are really great, removing condescending language really helps our docs be more friendly for the readers.

I've made a couple of suggestions. Fix those and we can approve it!

**Review by lukeocodes — APPROVED** [review]

**Comment by fabianrbz** [comment]
@mananchawla2005 thanks for your contribution!

**Comment by mananchawla2005** [comment]
@fabianrbz welcome uh kindly add hacktoberfest accepted tag not the hactober tag as mentioned by hf new rules. 
Thanks in advance

**Comment by lukeocodes** [comment]
oh that's right, the repo isn't opt-in specifically so it will need `hacktoberfest-accepted` as a tag 😂

**Comment by mananchawla2005** [comment]
> oh that's right, the repo isn't opt-in specifically so it will need `hacktoberfest-accepted` as a tag 😂 

Yeah  :-)

**Comment by mananchawla2005** [comment]
@fabianrbz thanks 🙂

**Comment by ruskibenya** [comment]
Hey @mananchawla2005, what a great contribution! Do you mind if we shout you out on Twitter?

**Pull request merged**